plugin/db/driver.go

```diff
@@ -332,12 +332,27 @@ type MigrationHistory struct {
 	Payload             string
 }
 
+// VersionFindOption is the option for version find.
+type VersionFindOption uint8
+
+const (
+	// Equal is the default option for version find.
+	Equal VersionFindOption = iota
+	GreaterThan
+	LessThan
+	NoGreaterThan
+	NoLessThan
+)
+
 // MigrationHistoryFind is the API message for finding migration historys.
 type MigrationHistoryFind struct {
 	ID *int
 
 	Database *string
-	Version  *string
+
+	Version           *string
+	VersionFindOption VersionFindOption
```

Comment:
If the only requirement for now is to fetch the most recent migration before versionXXX,

I would prefer to just have a version field and comment the behavior

"If both version and limit presents" then it will fetch the "Limit" most recent versions including "version" if exists.